proc-macros/src/rpc_macro.rs

```diff
@@ -81,11 +82,18 @@ impl RpcMethod {
 			.filter_map(|arg| match arg {
 				syn::FnArg::Receiver(_) => None,
 				syn::FnArg::Typed(arg) => match *arg.pat {
-					syn::Pat::Ident(name) => Some((name, *arg.ty)),
-					_ => panic!("Identifier in signature must be an ident"),
+					syn::Pat::Ident(name) => Some(Ok((name, *arg.ty))),
+					syn::Pat::Wild(_) => Some(Err(syn::Error::new(
+						span,
```

Comment:
Couldn't we get the span for the token itself here so the highlight actually shows what the problem is?

Reply:
Absolutely, fixed.